buildpipeline/dotnet-standard-win.json

```diff
@@ -79,15 +87,38 @@
         "nuGetAdditionalArgs": "-Timeout 3600",
         "verbosity": "Detailed",
         "nuGetVersion": "3.5.0.1829",
-        "nuGetPath": ""
+        "nuGetPath": "",
+        "continueOnEmptyNupkgMatch": "false"
+      }
+    },
+    {
+      "environment": {},
+      "enabled": true,
+      "continueOnError": false,
+      "alwaysRun": false,
+      "displayName": "packages -> Azure Feed",
+      "timeoutInMinutes": 0,
+      "refName": "CmdLine1",
+      "task": {
+        "id": "d9bafed4-0b18-4f58-968d-86655b4d2ce9",
+        "versionSpec": "1.*",
+        "definitionType": "task"
+      },
+      "inputs": {
+        "filename": "build.cmd",
+        "arguments": "-- publish.proj /t:PublishToAzureBlobFeed /p:AccountKey=$(PB_PublishAccountKey) /p:FeedUrl=$(PB_AzureFeedUrl) /p:SymbolFeedUrl=$(PB_AzureSymbolFeedUrl) /p:PublishSymbols=$(PB_PublishSymbols)",
```

Comment:
Every other repo calls this ExpectedFeedUrl instead of FeedUrl we should try to be consistent.